src/compiler/scala/tools/nsc/backend/jvm/BCodeIdiomatic.scala

```diff
@@ -40,7 +40,7 @@ abstract class BCodeIdiomatic extends SubComponent {
     if (emitStackMapFrame) asm.ClassWriter.COMPUTE_FRAMES else 0
   )
 
-  val StringBuilderClassName = "scala/collection/mutable/StringBuilder"
+  val JavaStringBuilderClassName = "java/lang/StringBuilder"
```

Comment:
I made it a `val` considering that it's highly unlikely that we don't need it, and probably need it more than once, if that's fine for you?